example/crd/federatedbar_crd.yaml

```diff
@@ -0,0 +1,106 @@
+#template type
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
```

Comment:
How about s/federatedbar_crd.yaml/federatedbar.yaml

As this file was already under `crd` folder

Reply:
The crd suffix shows that it contains CRD definition, opposite to creation of CRD instances.